ui/src/components/VolumeListTable.js

```diff
@@ -96,6 +96,10 @@ const ActionContainer = styled.span`
   padding: ${padding.large} ${padding.base} 26px 20px;
 `;
 
+const NameLinkContaner = styled.div`
+  cursor: pointer;
```

Comment:
1. We should remove the underline of the volume name.
2. We should add a margin between the volume name and usage progressbar.